src/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(CMAKE_BUILD_TYPE "${CMAKE_BUILD_TYPE}" CACHE STRING
 	FORCE
 )
 
+set(CMAKE_C_FLAGS "${CMAKE_CFLAGS}  $ENV{CFLAGS} $ENV{CPPFLAGS}")
```

Comment:
Fragile. Each time CMake runs, it rereads these environment variables.
Also, `CFLAGS` is in `CMAKE_C_FLAGS` already, isn’t it?